forum/qa-theme/SnowFlat/js/selectLanguage.js

```diff
@@ -0,0 +1,111 @@
+if (typeof CKEDITOR != 'undefined') {

```

Comment:
Dlaczego całość jest w global scope? Ten plik podpinasz do <haed> czy na koniec <body>?

Reply:
Plik podpinam do head

Reply:
To powinieneś owinąć kod w zdarzenie 'DOMContentLoaded', albo przesuń na koniec tagu BODY.